uvod-do-programovani-2/slicing-metody-moduly/excs/prijimacky.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Přijímačky a moduly
+demand: 2
+---
+
+Vrať se k příkladu o [přijímacích zkouškách](https://kodim.cz/kurzy/uvod-do-progr-2/uvod-do-programovani-2/uvod/priklady#excs%3Eprijimacky). V něm jsme počítali ručně průměr pomocí sčítání a dělení. Uprav své řešení (nebo vytvoř nové s využítím dvourozměrného seznam v zadání odkazovaného příkladu), které využije modul `statstics`. Na začátku vytvoř prázdný seznam a následně pomocí cyklu vlož do nového seznamu známky z předmětů, které jsou pro danou školu důležité (tj. český jazyk, anglický jazyk, matematika, fyzika). Následně použij metodu `statistics.mean()` k výpočtu průměru. Dále zkus využít funkce, které jsou zmíněné v textu, k výpočtu nejlepší a nejhorší známky z daných předmětů.
```

Comment:
Ja bych se nejradeji vyhnul linkovani (absolutnimu i relativnimu), protoze v dalsi verzi kodim se to klidne muze rozbit.

Zadani bych tam prevypravel a vstupni data zkopiroval.